lib/std/pdb.zig

```diff
@@ -399,17 +399,14 @@ pub const LineBlockFragmentHeader = extern struct {
 pub const LineNumberEntry = extern struct {
     /// Offset to start of code bytes for line number
     Offset: u32,
-    Flags: u32,
-
-    /// TODO runtime crash when I make the actual type of Flags this
-    pub const Flags = packed struct {
```

Comment:
Okay. I wasn't sure about renaming the fields here because I assumed they're matching the PDB spec / canonical impl. Looking at this file, it appears to generally avoid namespacing stuff like this; e.g. we have `pdb.LineFlags` rather than `pdb.LineFragmentHeader.Flags`. I could follow the same pattern here, so as to still expose the type without the name conflict?

Alternatively, I could just go ahead and rename every field in this file to follow naming conventions -- I think they probably *should* to be honest, even if the spec uses `TitleCase` field names. If you're happy with that solution, I'll go ahead and do it, keeping all the type namespacing as today for now.